### PR TITLE
Navbar Bug Fixed!

### PR DESCRIPTION
**Pull request by vbvh98**
# Changed navbar color on toggle
**header is transpatent when opened at top of page makes it difficult to view menu.**
> this commit fixes the issue.

Main changes:
- click handler added to .nav-toggler to toggle scorlled and dark classes on $nav and $text when color of header is transparent to make menu readable and consistent with UI.

Other changes:
- indentation used 2 spaces.
- all CSS selectors end with the new line, if comma seperated newline after each comma.
- moved transition property from .fixed-top.scrolled to .fixed-top for to add transition to both open and close of menu.
- 80 character wordwrap.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
-    $(document).scroll(function () {
-        var $nav = $(".fixed-top");
-        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height());
-        var $text = $(".text-white");
+$(function() {
+    $(document).scroll(function() {
+        var $nav = $('.fixed-top')
+        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height())
+        var $text = $('.text-white')
         $text.toggleClass('dark', $(this).scrollTop() > $nav.height())
```
Comment (mooocer) [inline js/script.js]: There should be space after the function name, and double quotes and single quotes can be used interchangeably, so please remove these changes everywhere.
https://eslint.org/docs/rules/space-after-function-name

File: js/script.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
-    });
-});
\ No newline at end of file
+    })
+    $('.navbar-toggler').click(() => {
+        var $nav = $('.fixed-top')
+        var $text = $('.text-white')
```
Comment (mooocer) [inline js/script.js]: The `.text-white` class will be used by many other elements. Will it be better to use an id to refer to the elements?

**Review by mooocer — CHANGES_REQUESTED** [review]
See the comments, and squash the commits into one.

File: js/script.js (hunk 1 above)
Comment (vbvh98) [reply js/script.js]: ok!

File: js/script.js (hunk 2 above)
Comment (vbvh98) [reply js/script.js]: yes

**Review by vbvh98 — COMMENTED** [review]
fixed code formating as per your review

**Comment by mooocer** [comment]
Hey @vbhv98 thanks, but I still see 10 commits here. I am sorry but we can't merge 10 commits into the repository. Please rebase your commits on the latest changes, squash your commits, and force push to your branch, so that the pull request gets updated.

**Comment by mooocer** [comment]
Dude don't keep closing pull requests unless you plan to open a new one. Update this one itself.

**Pull request closed without merging**